hail/src/main/scala/is/hail/expr/ir/streams/EmitStream.scala

```diff
@@ -468,9 +468,12 @@ object EmitStream {
 
               override val LproduceElement: CodeLabel = mb.defineAndImplementLabel { cb =>
                 val Lfiltered = CodeLabel()
+                cb.define(Lfiltered)
```

Comment:
no bug here, just a clearer implementation

Reply:
it gets cleared every time we jump to LProduceElement. But after some time away, I think you're right - this is worse.